src/components/FlightDetails.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from "react";
+import { useParams, useNavigate } from "react-router-dom";
+import {
+  getCurrentTime,
+  getTimeFromDateString,
+  getStatusColor,
+} from "../utility";
+import useFlightApi from "../Apis";
+
+const FlightDetails = () => {
+  const history = useNavigate();
+  const [flightDetails, setFlightDetails] = useState([]);
+  const [errorApi, setErrorApi] = useState(null); // State to store error information
+  const { id } = useParams();
+
+  const { data, error = null } = useFlightApi(
+    `https://flight-status-mock.core.travelopia.cloud/flights/${id}`
+  );
+
+  useEffect(() => {
+    setFlightDetails(data);
+    setErrorApi(error);
+  }, [data, error]); // Include 'id' in dependency array to fetch data when 'id' changes
+
+  const navigateFlightList = () => {
+    history("/");
+  };
+
+  return (
+    <div className="bg-container">
+      <div className="container-main">
+        {errorApi ? (
+          <p>There was an error: {errorApi}</p>
+        ) : (
+          <div className="detailParent-container">
+            {/* flight heading */}
+            <p className="back-btn" onClick={navigateFlightList}>
+              Back
+            </p>
+            <div className="flightDetails-header">
```

Comment:
there can't be two headers for flightDetails. either change the class name, or create a different class. 

There's flightDetails-header here and below as well.

Reply:
changes class name